tests/conftest.py

```diff
@@ -1,10 +1,10 @@
-import email
 import pytest
 from fastapi.testclient import TestClient
+from sqlalchemy.orm import Session
+from bcrypt import hashpw, gensalt
```

Comment:
It should be `bcrypt < fastapi < sqlalchemy` since `b < f < s` in lexicographical order.